src/wp-content/themes/twentynineteen/functions.php

```diff
@@ -288,6 +288,49 @@ function twentynineteen_scripts() {
 }
 add_action( 'wp_enqueue_scripts', 'twentynineteen_scripts' );
 
+function twentynineteen_generate_color_palette_css() {
+
+	// Check if the theme supports editor color palette.
+	if ( ! current_theme_supports( 'editor-color-palette' ) ) {
+		return;
+	}
+
+	$colors = get_theme_support( 'editor-color-palette' );
+	if ( ! $colors ) {
+		return;
+	}
+
+	$colors = $colors[0];
+	$css    = '';
+
+	foreach ( $colors as $color ) {
+		// Generate text color classes
```

Comment:
```suggestion
		// Generate text color classes.
```

Fix for other inline comment

Reply:
Fixed for all inline comments